olclient/entity_helpers/work.py

```diff
@@ -0,0 +1,172 @@
+from __future__ import annotations
```

Comment:
Currently we would need this so that methods inside `Work` class can return back the same class `Work`. For example, https://github.com/internetarchive/openlibrary-client/pull/235/files#diff-1f805721a67602b2435d9b8eef273543ee2aa00b4a92dbcf5ffee7e2de7489a9R91

Reply:
Oh interesting! I guess we're not in the future yet :)